pkg/backup/request.go

```diff
@@ -11,6 +11,7 @@ import (
 type Request struct {
 	*velerov1api.Backup
 
+	PodVolumeBackups          []*velerov1api.PodVolumeBackup
```

Comment:
Let's move this down to be grouped with `VolumeSnapshots` since the top group is all "spec"-related fields.